routes.js

```diff
@@ -0,0 +1,102 @@
+var Todo = require('./models/todo')(Todo);
+
+module.exports = (server) => {
+
+    function errorMessage(testName, error) {
+        console.log("There is an error with " + testName + " : " + error);
+    }
+    function objectNotFound(res, id){
+        res.send('object with id ' + id + ' does not exist.');
+    }
+    //Test 1: Create a Todo
+    server.post('/todos', async (req, res, next) => {
+        try {
+            const listObject = await Todo.create({
```

Comment:
I think I found having two todos potentially confusing, but I think I agree in the end.